flask_rebar/rebar.py

```diff
@@ -459,9 +462,18 @@ def add_handler(
         :param Type[USE_DEFAULT]|None|str mimetype:
             Content-Type header to add to the response schema
         """
-        if isinstance(response_body_schema, marshmallow.Schema):
+        if isinstance(response_body_schema, marshmallow.Schema) or isclass(
+            response_body_schema
+        ):
             response_body_schema = {200: response_body_schema}
 
+        # Fix #115: if we were passed bare classes we'll go ahead and try to instantiate
+        request_body_schema = normalize_schema(request_body_schema)
+        query_string_schema = normalize_schema(query_string_schema)
+        if response_body_schema:
+            for code, schema in response_body_schema.items():
```

Comment:
```suggestion
            response_body_schema = {code: normalize_schema(schema) for (code, schema) in response_body_schema.items()}
```
clearer as a dict comprehension